src/components/input/template.test.js

```diff
@@ -0,0 +1,120 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('input')
+
+describe('Input', () => {
+  describe('by default', () => {
+    it('renders with classes', () => {
+      const { $ } = render('input', {
+        classes: 'app-c-input--custom-modifier'
+      })
+
+      const $component = $('.govuk-c-input')
+      expect($component.hasClass('app-c-input--custom-modifier')).toBeTruthy()
+    })
+
+    it('renders with id', () => {
+      const { $ } = render('input', {
+        id: 'my-input',
+        name: 'my-input-name'
```

Comment:
We don't test for name, and given that in the example above we omit both id and name, do we want to do the same here and just provide an `id`? Conversely, if we're providing `name` and `id` in the other examples because they are 'required', should we include them in the classes example also?

Reply:
Indeed I started to add `id` and `name` because they are 'required', but I'm not sure it's the right approach and right now is inconsistent across examples. I would rather add the minimum number of attributes since we cannot really enforce the 'required' state.

Reply:
Removed unused attributes for now, even if defined as 'required'.